docker-compose.yml

```diff
@@ -0,0 +1 @@
+./compose/balena.yml
```

Comment:
Do you know why this file is the way it is?
Can you even do that, just have the path referencing some other file?

Reply:
I don't understand, locally the file doesn't look like that..

Reply:
@rcooke-warwick  How does it look like locally?
Also is it because of this https://github.com/balena-os/leviathan/blob/b9e7f9f3478376ffda3cfc5c0ea970a292fe9881/.gitignore#L8

Reply:
Okay pulled the branch and saw what you mean. Weird indeed.